.pre-commit-config.yaml

```diff
@@ -33,13 +25,19 @@ repos:
     hooks:
       - id: gen-schema
         name: gen-schema
-        entry: ./hooks/gen_schema.sh
+        entry: python hooks/gen_schema.py
         language: system
+        types: [python]
         require_serial: true
-  - repo: local
-    hooks:
       - id: gen-docs
         name: gen-docs
-        entry: ./hooks/gen_docs.sh
+        entry: python hooks/gen_docs.py
         language: system
+        types: [python]
         require_serial: true
+      - id: mypy
```

Comment:
any reason for changing the order of execution?

Reply:
Not really, it wasn't intentional, will change it back